desktop-tool/src/constants.py

```diff
@@ -172,7 +172,17 @@ class TargetSites(Enum):
             Cardstocks.P10: "Kunststoff",
         },
     )
-
+    PrinterStudioFR = TargetSite(
+        base_url="https://www.printerstudio.fr",
+        starting_url_route="personnalise/cartes-de-jeu-sur-mesure-cartes-blanches.html",
+        supports_foil=False,
+        cardstock_site_name_mapping={
```

Comment:
`saved_successfully_text` must be specified (it will default to the english text otherwise):
```suggestion
        saved_successfully_text="Enregistré avec succès",
        cardstock_site_name_mapping={
```